### PR TITLE
Update to 1.20.1

### PR DESCRIPTION
**Pull request by EvieTheOwl**
I like ur mod and wanted to use it in an updated modpack so I've updated it. I hope that's okay with you. No new functionality.

# Changes
- Removed accesswidener. Both entries are not needed now 
- Fixed frozen bomb permanently causing a person to have the ice effect. (Might be to do with this version)
- Did 1.20 things like moving damage tyes into the resources folder and implementing dynamic item groups

### CODE REVIEW

**Comment by Pyrofab** [comment]
Could you pull in changes from #33 ?

File: build.gradle [hunk 1]
```diff
@@ -1,5 +1,5 @@
 plugins {
-	id 'fabric-loom' version '0.11-SNAPSHOT'
+	id 'fabric-loom' version '1.1-SNAPSHOT'
 	id 'io.github.ladysnake.chenille' version '0.9.0'
```
Comment (Pyrofab) [inline build.gradle:3]:
```suggestion
	id 'io.github.ladysnake.chenille' version '0.11.3'
```

File: src/main/java/ladysnake/blast/client/particle/ConfettiParticle.java [hunk 2]
```diff
@@ -118,14 +120,14 @@ public void tick() {
         if (this.age++ >= this.maxAge) {
             this.markDead();
         } else {
-            if (this.world.getFluidState(new BlockPos(this.x, this.y + 0.2, this.z)).isEmpty()) {
-                if (this.world.getFluidState(new BlockPos(this.x, this.y - 0.01, this.z)).isIn(FluidTags.WATER)) {
+            if (this.world.getFluidState(new BlockPos((int) this.x, (int) (this.y + 0.2), (int) this.z)).isEmpty()) {
+                if (this.world.getFluidState(new BlockPos((int) this.x, (int) (this.y - 0.01), (int) this.z)).isIn(FluidTags.WATER)) {
```
Comment (Pyrofab) [inline src/main/java/ladysnake/blast/client/particle/ConfettiParticle.java]: I believe the right method to use here would be `BlockPos.ofFloored` (also applies to all the other BlockPos int casts)

File: src/main/java/ladysnake/blast/common/block/StripminerBlock.java [hunk 3]
```diff
@@ -128,9 +128,8 @@ public ActionResult onUse(BlockState state, World world, BlockPos pos, PlayerEnt
             world.setBlockState(pos, Blocks.AIR.getDefaultState(), 11);
             if (!player.isCreative()) {
                 if (item == Items.FLINT_AND_STEEL) {
-                    itemStack.damage(1, player, (playerEntity) -> {
-                        playerEntity.sendToolBreakStatus(hand);
-                    });
+                    itemStack.damage(1, player, (playerEntity)
+                            -> playerEntity.sendToolBreakStatus(hand));
```
Comment (Pyrofab) [inline src/main/java/ladysnake/blast/common/block/StripminerBlock.java]: Maybe put this on a single line if you remove the braces ?

**Comment by EvieTheOwl** [comment]
Notes on **Fixing commit merge mistakes**. I merged the 1.20 version thinking it was the newest one but later realised after testing I should have merged 1.19 which had the changes aaaaa. Fixed it manually though and tested every bomb individually.

I also downgraded chenille to 0.9.0 since newer versions require java 19 which it really shouldn't.

**Comment by EvieTheOwl** [comment]
Finished :3 @Pyrofab

**Comment by Pyrofab** [comment]
> I also downgraded chenille to 0.9.0 since newer versions require java 19 which it really shouldn't.

Darn, I must have misconfigured the buildscript again

**Review by Pyrofab — APPROVED** [review]
Excellent work !

**Pull request merged**